scripts/seedData.ts

```diff
@@ -0,0 +1,207 @@
+import contentfulManagement, { Environment } from 'contentful-management';
+import fs from 'fs';
+import { join, dirname } from 'path';
+import { fileURLToPath } from 'url';
+import * as dotenv from 'dotenv';
+
+type GenreMap = {
+	[genre: string]: string;
+};
+
+type Movie = {
+	title: string;
+	posterURL: string;
+	summary: string;
+	duration: string;
+	director: string[];
+	mainActors: string[];
+	genres: string[];
+	datePublished: string;
+	rating: string;
+	ratingValue: number;
+	bestRating: number;
+	worstRating: number;
+	writers: string[];
+};
+
+type Sys = {
+	id: string;
+	linkType: string;
+	type: string;
+};
+
+type MovieGenres = Record<string, { sys: Sys }[]>;
+
+// load dotenv config
+dotenv.config();
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
```

Comment:
Why are you naming your variables this way?

Reply:
I found it as a way to replicate the functionality of __filename and __dirname in ES modules.

Reply:
That doesn't make sense to me.

Reply:
AFAIK `__filename` and `__dirname` are not globally available in ES modules. I named those variables like that because they replicate their functionality

Reply:
```suggestion
const pathToThisFile = fileURLToPath(import.meta.url);
const seedFileDir = dirname(__filename);
```